backend/data_access/storage/postgres_store.py

```
@@ -242,27 +323,26 @@ def store_vol_surface(self, vol_surface: VolSurface) -> int:
                     points_data,
                     page_size=100,
                 )
 
                 self.conn.commit()
-                return vol_surface_id
+                return surface_id
 
         except Exception as e:
             self.conn.rollback()
-            raise RuntimeError(f"Failed to store vol surface: {str(e)}")
+            raise RuntimeError(f"Failed to store surface: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)